### PR TITLE
About fixes

### PR DESCRIPTION
**Pull request by mirshko**
addresses #28

### CODE REVIEW

File: src/pages/about.js [hunk 1]
```diff
@@ -128,7 +128,8 @@ const AboutPage = ({ data }) => {
 
         {/* Block 7 */}
         <Grid justify="center">
-          <Column textAlign="center">
+          <Spacer width={3} />
+          <Column span={{ medium: 12, large: 6 }} textAlign="center">
```
Comment (rdinicut) [inline src/pages/about.js]: IMO you should not pass breaking point styles as props in JSX but handle that in the theme. Let's say you want to add a another breaking point. You will need to ajust the props on every Column component in the app. How many Grids you have that have different different spans for for the medium breaking point for example?

Comment (mirshko) [reply src/pages/about.js]: basically a lot. almost all of them.

Comment (mirshko) [reply src/pages/about.js]: dunno if there is a good way to build these layouts unless making a few components for 2 column 3 column 4 column etc

Comment (mirshko) [reply src/pages/about.js]: bit late to go back and rework all this. prob should have used something like https://flexgridlite.elliotdahl.com/ or whatever

File: src/components/Grid/index.js [hunk 2]
```diff
@@ -14,16 +16,19 @@ const GridWrapper = styled(Box)`
 `;
 
 const StyledGrid = styled.section`
-  display: flex;
-  flex-direction: column;
+  display: grid;
+  grid-template-columns: repeat(12, 1fr);
+  grid-column-gap: 32px;
```
[the remark this answers is not in the document]
Comment (mirshko) [reply src/components/Grid/index.js]: will eventually refactor this into the native grommet grid.

**Pull request merged**